src/einsteinpy/tests/test_symbolic/test_schouten.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import sympy
+from sympy import cos, sin, sinh
+
+from einsteinpy.symbolic import (
+    MetricTensor,
+    RicciScalar,
+    RicciTensor,
+    RiemannCurvatureTensor,
+    SchoutenTensor,
+)
+
+
+def anti_de_sitter_metric():
+    coords = sympy.symbols("t chi theta phi")
+    t, ch, th, ph = coords
+    m = sympy.diag(
+        -1,
+        cos(t) ** 2,
+        cos(t) ** 2 * sinh(ch) ** 2,
+        cos(t) ** 2 * sinh(ch) ** 2 * sin(th) ** 2,
+    ).tolist()
+    metric = MetricTensor(m, coords)
+    return metric
+
+
+def schwarzschild_metric():
+    symbolstr = "t r theta phi"
+    syms = sympy.symbols(symbolstr)
+    G, M, c, a = sympy.symbols("G M c a")
+    # using metric values of schwarschild space-time
+    # a is schwarzschild radius
+    list2d = np.zeros((4, 4), dtype=int).tolist()
+    list2d[0][0] = 1 - (a / syms[1])
+    list2d[1][1] = -1 / ((1 - (a / syms[1])) * (c ** 2))
+    list2d[2][2] = -1 * (syms[1] ** 2) / (c ** 2)
+    list2d[3][3] = -1 * (syms[1] ** 2) * (sympy.sin(syms[2]) ** 2) / (c ** 2)
+    sch = MetricTensor(list2d, syms)
+    # print (sch.tensor())
+    return sch
+
+
+def test_SchoutenTensor_parent_metric_property_and_ValueError():
+    mt = anti_de_sitter_metric()
+    St = SchoutenTensor.from_metric(mt)
+    assert St.parent_metric == St._parent_metric and St.parent_metric == mt
+    # ValueError test part
+    try:
+        St2 = SchoutenTensor(St.tensor(), St.syms, config="uuu")
+        assert False
+    except ValueError:
+        assert True
+
+
+def test_SchoutenTensor():
+    metric = anti_de_sitter_metric()
+    st = SchoutenTensor.from_metric(metric)
```

Comment:
What's this test for??. 
I don't find it useful. Maybe I am wrong.
Can you explain it??

Reply:
I did it to see that it called the function well, but now it is no longer useful. I'm going to delete it